lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -275,6 +275,11 @@
     @include idashbutton(#eee);
   }
 
+  select {
+      border: 1px solid $light-gray;
+      height: 34px;
```

Comment:
Should we really be hardcoding the height? How will it look over different orientations?

Reply:
@DawoudSheraz I checked the orientation of the page on different scales, it looks good. Actually, the button on the page has a height of 34px.